pkgs/applications/audio/zrythm/default.nix

```
@@ -161,13 +192,17 @@ stdenv.mkDerivation rec {
     gappsWrapperArgs+=(
       --prefix GSETTINGS_SCHEMA_DIR : "$out/share/gsettings-schemas/${pname}-${version}/glib-2.0/schemas/"
       --prefix XDG_DATA_DIRS : "$XDG_ICON_DIRS"
-             )
+      --prefix XDG_DATA_DIRS : "${breeze-icons}/share"
+    )
   '';
 
   meta = with lib; {
     homepage = "https://www.zrythm.org";
-    description = "Highly automated and intuitive digital audio workstation";
-    maintainers = with maintainers; [ tshaynik magnetophon ];
+    description = "Automated and intuitive digital audio workstation";
+    maintainers = with maintainers; [
+      tshaynik
+      magnetophon
+    ];
```

```suggestion
    maintainers = with maintainers; [ tshaynik magnetophon ];
```
nit
we don't need to change this. Doing it like this is fine.